diff --git a/res/field/scripts/scripts_route_201.s b/res/field/scripts/scripts_route_201.s
index 52794930..36235015 100644
--- a/res/field/scripts/scripts_route_201.s
+++ b/res/field/scripts/scripts_route_201.s
@@ -285,12 +285,9 @@ Route201_Briefcase:
     GiveMysteryStarterEgg
     Message Route201_Text_TheEggIsHatching
     WaitABPress
-    FadeScreenOut
-    WaitFadeScreen
     CloseMessage
-    HatchMysteryStarterEgg
-    FadeScreenIn
-    WaitFadeScreen
+    GetPlayerStarterSpecies VAR_0x8000
+    GivePokemon VAR_0x8000, 5, ITEM_NONE, VAR_RESULT
     ApplyMovement LOCALID_PROF_ROWAN, Route201_Movement_ProfRowanFacePlayerSouth
     ApplyMovement LOCALID_RIVAL, Route201_Movement_RivalFaceWest
     ApplyMovement LOCALID_PLAYER, Route201_Movement_PlayerFaceProfRowanNorth
diff --git a/res/text/route_201.json b/res/text/route_201.json
index 91e0b53f..0a25d9cd 100644
--- a/res/text/route_201.json
+++ b/res/text/route_201.json
@@ -317,8 +317,8 @@
     {
       "id": "Route201_Text_TheEggIsHatching",
       "en_US": [
-        "Oh?\n",
-        "The Egg is hatching!\r"
+        "DIAG: EGG CREATED\n",
+        "(no hatch)\r"
       ]
     },
     {
